src/components/data/data-utils.js

```diff
@@ -326,7 +328,8 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
       await dataProvider.uploadTsv({ workspaceId, recordType, file, useFireCloudDataModel, deleteEmptyValues, namespace, name })
       onSuccess()
       Ajax().Metrics.captureEvent(Events.workspaceDataUpload, {
-        workspaceNamespace: namespace, workspaceName: name, providerName: dataProvider.providerName
+        workspaceNamespace: namespace, workspaceName: name, providerName: dataProvider.providerName,
+        cloudPlatform: dataProvider.providerName === wdsProviderName ? cloudProviders.azure.label : cloudProviders.gcp.label
```

Comment:
Just trying to understand what this is doing - if the providerName is WDS, then that means it's azure, otherwise GCP?  So sometime in the distant future when WDS is on GCP we'd need to update this logic?

Reply:
yes exactly - I alluded to that in the PR description, but I think for now it's fine to just log both properties